websites/L/Los Angeles Times/presence.ts

```diff
@@ -0,0 +1,204 @@
+const presence = new Presence({
+		clientId: "1005501034191409153",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const title = document.title.replace(" - Los Angeles Times", ""),
+		[privacyMode, showButtons, showArticleAuthor, showMoreDetails] =
+			await Promise.all([
+				presence.getSetting<boolean>("privacy"),
+				presence.getSetting<boolean>("buttons"),
+				presence.getSetting<boolean>("articleAuthor"),
+				presence.getSetting<boolean>("moreDetails"),
+			]),
+		{ pathname, href, search } = window.location,
+		path = pathname.split("/"),
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp,
+		};
+
+	if (showButtons && !privacyMode) {
+		presenceData.buttons = [
+			{
+				label: "View Page",
+				url: href,
+			},
+		];
+	}
```

Comment:
You could remove this check and just place one at the bottom to delete the buttons if the button setting is false?
```suggestion
		presenceData.buttons = [
			{
				label: "View Page",
				url: href,
			},
		];
```

Reply:
Although it is more repetitive, the former method of doing so is more preformant.